### ISSUE TITLE
How to support redirects or multiple paths?

### ISSUE THREAD

**Issue by tscanlin** [MoOx/phenomic#773]
What would you recommend as the best way to handle redirects or multiple paths for a given page?

**Comment by tscanlin** [comment]
Love the project idea btw :)

**Comment by DavidWells** [comment]
This might not apply to your project but I'm using https://www.netlify.com/docs/redirects/ to handle redirects at the hosting level

**Comment by tscanlin** [comment]
That's cool that netlify supports it. I just thought it would be a useful feature for people transitioning from other sitemaps so that transitioning is easier and links don't die. In a SPA for instance it should be very easy and useful to support redirects on the client side. And if the site is purely static HTML on github pages for instance you can't modify server configs so doing redirects with extra copies of pages or a page with the meta attribute eg: `<meta http-equiv="refresh" content="0;URL='http://thetudors.example.com/'" />` 

This might be out of scope for phenomic core, but would it be possible from a plugin?

**Comment by MoOx** [comment]
Dup of #628. Will definitely be in core.

tl;dr: we have to handle it during prerendering, there is a wip
